main.js

```diff
@@ -0,0 +1,75 @@
+var button = document.querySelector('button');
+var input = document.querySelector('input');
+var h1 = document.querySelector('h1');
+
+
+// button adds event and starts timer
+
+var addText = function (){
+  // create <p>
+  var p = document.createElement('p');
+  // store value from input
+  var storeInput = input.value;
+  p.textContent = "Currently working on..." + storeInput;
+
+var body = document.querySelector('body');
+  body.appendChild(p);
+
+};
+
+button.addEventListener('click', addText);
+
+
+// add counter
+
+var number = 25;
+var intervalID;
+
+
+var countUp = function(){
+  h1.textContent = number;
+  number = number - 1;
+  return number;
+}
+
+
+// starts with button
+button.addEventListener('click', function(evt){
+  intervalID = setInterval(countUp, 1000);
```

Comment:
Nice job adding the listener to countdown, however you'll see this is running every second subtracting one from 25, but we're interested in creating a timer that goes for 25 minutes or 25 * 60 * 1000 milliseconds...